uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -927,22 +847,44 @@ public function upgrade_1037() {
    * @return bool
    */
   public function upgrade_1038() {
-    $optionValue = civicrm_api3('OptionValue', 'get', [
-      'option_group_id' => 'ta_settings',
-      'name' => 'days_to_create_a_document_clone',
-    ]);
-    if(empty($optionValue['id'])) {
+    // NOOP
+    // This is now done by the postInstall hook
+
+    return TRUE;
+  }
+
+  /**
+   * Migrate the T&A Settings from Option Values to CiviCRM Settings.
+   *
+   * As part of the migration, the Option Values (and their Option Group)
+   * are deleted.
+   *
+   * @return bool
+   * @throws \CiviCRM_API3_Exception
```

Comment:
This came up before, but are we encouraging the use of `@throws`?

Reply:
Personally, it's not a big deal for me if we encourage it or not, but I tend to think that we should use it. We haven't agreed on this though, so, for now, I think it's ok if people add it or not